WooCommerce/Classes/POS/Presentation/ItemListView.swift

```diff
@@ -84,26 +84,20 @@ struct ItemListView: View {
         VStack(spacing: 0) {
             headerView
 
-            if shouldShowSearchField && searchTerm.isEmpty {
-                POSRecentSearchesView(
-                    savedSearches: posModel.searchHistory(for: selectedItemListType.itemType),
-                    onSearchSelected: { search in
-                        searchTerm = search
-                    }
-                )
-                .background(Color.posSurface)
-            } else {
-                switch itemListState {
-                case .loading(let items),
-                        .loaded(let items, _),
-                        .inlineError(let items, _, _):
-                    listView(items)
-                case .error(let errorState):
-                    errorView(errorState)
-                case .empty:
-                    emptyView
-                }
-            }
+            TabView(selection: $selectedItemListType) {
+                itemListContent
+                    .tag(ItemListType.products(search: false))
+                    .gesture(DragGesture())
+                itemListContent
+                    .tag(ItemListType.products(search: true))
+                    .gesture(DragGesture())
```

Comment:
If you remove these three lines, I think the search/drag issue is solved.

You can't drag from search to other tabs (which makes sense) and you don't activate search by dragging.

Reply:
It could be related to `DashboardView` holding the state of selectedItem and search term. Within the body, we're switching over `itemsViewState.containerState`. When we enter a search term, the `itemViewState` changes to loading and then to `content`. Maybe these changes cause the redraw. When we simply change the selected item, these kind of state changes are not happening.

Reply:
I think DashboardView shouldn't be dependent on `selectedItemListType`, this is a state of the specific item type, which is handled within `ItemListView`.

Reply:
I'll experiment with moving `headerView` and `tabView` to `DashboardView`. It's easier than the reverse, since we would need to refactor how we handle container states.

Reply:
Sorry, I didn't see your earlier comment for some reason – the notification only came through a moment ago.

DashboardView mainly depends on it for historical reasons of the container state... if that were separated out, or we stopped doing fullscreen loading/errors, it could perhaps not be referenced there. 

But it still needs to be higher up the stack than ItemListView so that it can survive going back and forth from the cart.